loom.config.ts

```diff
@@ -1,15 +1,18 @@
 import {createWorkspace, createWorkspaceTestPlugin} from '@shopify/loom';
+import {
+  buildLibrary,
+  buildLibraryWorkspace,
+} from '@shopify/loom-plugin-build-library';
 import {eslint} from '@shopify/loom-plugin-eslint';
-import {jest} from '@shopify/loom-plugin-jest';
 import {prettier} from '@shopify/loom-plugin-prettier';
-import {workspaceTypeScript} from '@shopify/loom-plugin-typescript';
+
+import type {} from '@shopify/loom-plugin-jest';
```

Comment:
Curious about why it's importing an empty object? Is this intentional?

Reply:
It is intentional, but it is not obvious why.

Loom does some kinda funky things with augmenting objects, that don't quite always work super cleanly.  
This is needed so that typescript realises that `hooks.jestConfig` is a valid key on the hooks object and that it has a particular shape of its own. Without this it thinks that jestConfig is not a valid key.

I'll add a comment explaining this

Reply:
Thanks for elaborating and for adding the comment 🎉